src/osd/PGTransaction.h

```diff
@@ -20,6 +20,7 @@
 
 #include "common/hobject.h"
 #include "osd/osd_types.h"
+#include "osd/osd_internal_types.h"
```

Comment:
This seems like a pretty obtuse way to get osd_internal_types where it needs to go. Maybe it's included here by happenstance but shouldn't we place it in the right classes?